PocketBeagle/Grove/Accelerometer.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2020 SeeedStudio
```

Comment:
Wrong directory for now.

Reply:
Can you move the copyright to the bottom or to a different file to make reading the examples easier?